tests/utils/test_data.py

```diff
@@ -576,6 +576,7 @@ def test_get_dataframe__from_csv_file_with_nulls_in_some_columns__set_non_na_col
             data[-2]['str_col'] = np.nan
             df = pd.DataFrame(data)
             df.to_csv(path_or_buf=fp, columns=df.columns, encoding='utf-8', index=False)
+            df['str_col'] = df['str_col'].map(lambda x: np.nan if x in PANDAS_DEFAULT_NULL_VALUES else x)
```

Comment:
Is there a reason why "`str_col`" here is lowercase?

Reply:
I'm not sure why it was lower case in this example, i followed the test which was using `data[-2]['str_col'] = np.nan` instead of `data[-2]['STR_COL'] = np.nan` like the others

Reply:
fair enough'